### PR TITLE
MAUI 843542 - Update code example for toolbar UG on image editor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
I have updated a code example for toolbar UG.

### CODE REVIEW

File: MAUI/ImageEditor/toolbar-customization.md [hunk 1]
```diff
@@ -247,12 +344,12 @@ The ImageEditor automatically generates the toolbars. To add the toolbars manual
                 <imageEditor:ImageEditorToolbarGroupItem Alignment="Start">
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/toolbar-customization.md:344]: Add note:

When you create a new toolbar item, you can use pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce new icons or modify the current one, you can use the `View` property of the `ImageEditorToolbarItem` class to define the item view.

File: MAUI/ImageEditor/toolbar-customization.md [hunk 2]
```diff
@@ -231,6 +229,105 @@ Change the default colors of the `ColorPalette` in the toolbar.
...
+
+{% endtabs %}
+
+### Customize the Toolbar Item View
+
+To customize the view of toolbar items, specify their icon names and set the view using the `View` property in the `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/toolbar-customization.md:262]: Change this code

SfImageEditor imageEditor = new SfImageEditor();
imageEditor.Source = ImageSource.FromFile("image.png");

ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");

Image image = new Image();
image.Source = ImageSource.FromFile("save.jpg");
saveItem.View = image;

File: MAUI/ImageEditor/toolbar-customization.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/toolbar-customization.md:346]: added.

File: MAUI/ImageEditor/toolbar-customization.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/toolbar-customization.md:262]: updated.

File: MAUI/ImageEditor/toolbar-customization.md [hunk 3]
```diff
@@ -231,10 +229,111 @@ Change the default colors of the `ColorPalette` in the toolbar.
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
 ## Explicitly Create Toolbars
 
 The ImageEditor automatically generates the toolbars. To add the toolbars manually, you need to explicitly add them and change the `AutoGenerateToolbarItems` property to `false`.
 
+N> When you create a new toolbar item, you can use pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce new icons or modify the current one, you can use the View property of the `ImageEditorToolbarItem` class to define the item view.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/toolbar-customization.md:335]: When you create a new toolbar item, you can use the pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce a new item view or modify the current one, you can use the `View` property of the `ImageEditorToolbarItem` class to define the item view.

Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/toolbar-customization.md]: updated.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**